app/resume/[username]/page.tsx

```diff
@@ -3,7 +3,28 @@ import { createServerComponentClient } from "@supabase/auth-helpers-nextjs";
 import React from "react";
 import { cookies } from "next/headers";
 import { redirect } from "next/navigation";
+import { Metadata } from "next";
 
+type Props = {
+  params: { username: string };
+  searchParams: { [key: string]: string | string[] | undefined };
+};
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+  return {
+    title: `git-re `,
+    description: `resume generated by git-re for ${params.username}`,
+    openGraph: {
+      images: [
+        {
+          url: `http://localhost:3000/api/og?username=${params.username}`,
```

Comment:
Why is it localhost?

Reply:
forgot to change here just a sec

Reply:
sure and resolve conflicts. In the video I could not the og-image and all. Can you make a video displaying that?

Reply:
I showed using the og-spy extension at the top-right as I was testing in local dev server the og-tag won't show but it will in deployment.